runtime/recipe/particle.js

```diff
@@ -86,6 +86,11 @@ class Particle {
   isResolved() {
     assert(Object.isFrozen(this));
     // TODO: slots
+    if (this.consumedSlotConnections.length > 0) {
+      let fulfilledSlotConnections = this.consumedSlotConnections.filter(connection => connection.targetSlot !== undefined);
```

Comment:
This is the test for at least one connection is fulfilled - it can't go in a single SlotConnection because it needs to look at all of them.